packages/nimble-components/src/table/tests/table-matrix.stories.ts

```diff
@@ -0,0 +1,41 @@
+import type { Meta, Story } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { html, ViewTemplate } from '@microsoft/fast-element';
+import {
+    createMatrixThemeStory,
+    createStory
+} from '../../utilities/tests/storybook';
+import {
+    createMatrix,
+    sharedMatrixParameters
+} from '../../utilities/tests/matrix';
+import { hiddenWrapper } from '../../utilities/tests/hidden';
+import '../../all-components';
+
+const metadata: Meta = {
+    title: 'Tests/Table',
+    decorators: [withXD],
+    parameters: {
+        ...sharedMatrixParameters(),
+        design: {
+            artboardUrl:
+                'https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/b9cee5e2-49a4-425a-9ed4-38b23ba2e313/specs/'
+        }
+    }
+};
+
+export default metadata;
+
+// prettier-ignore
+const component = (
+): ViewTemplate => html`
+    <nimble-table></nimble-table>
+`;
+
+export const tableThemeMatrix: Story = createMatrixThemeStory(
+    createMatrix(component)
+);
+
+export const hiddenTable: Story = createStory(
```

Comment:
Many of our components also include a "Text customized" story that ensures that setting the component's font to be italics has an effect on rendering. I'm on the fence about including it now for the table too. Thoughts?

My initial thought: It seems unlikely a client would want the entire table contents to be italics. Rather they'd want to style individual cells or headers. This implies we'd probably have to do feature dev to make this work correctly, so we can wait until that's prioritized to add this test.